src/com/luxoft/java/course/shevtsov/homework/algorithm/additionals/QuickSort_NoRecursoin.java

```diff
@@ -0,0 +1,50 @@
+package com.luxoft.java.course.shevtsov.homework.algorithm.additionals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class QuickSort_NoRecursoin {
```

Comment:
Do not use any characters except letters in name

Reply:
Should I use this rule for all names (classes, variables, methods, etc.)?

Reply:
http://web.archive.org/web/20140228225807/http://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Reply:
cought it. thanks